### DISCUSSION TITLE
Has anyone tried to deployed on shared hosting

### DISCUSSION THREAD

**Discussion by mikaoelitiana** [pretix/pretix#4614] [Q&A]
Is it possible to deploy pretix on a shared hosting like those used to run Wordpress? Most web hosting server have CPanel that supports deploying Python apps (cf. https://support.cpanel.net/hc/en-us/articles/360049921014-How-do-I-set-up-a-Python-web-application). They also often have Postgres, redis and memcached as well.

**Comment by raphaelm** [comment]
If they have all that, it should be possible. We haven't tried. This might be the closest thing: https://lab.uberspace.de/guide_pretix/
